pom.xml

```diff
@@ -25,7 +25,7 @@
 
     <groupId>com.connexta.search</groupId>
     <artifactId>query-api</artifactId>
-    <version>0.3.1-SNAPSHOT</version>
+    <version>0.4.0-SNAPSHOT</version>
```

Comment:
❓ Does the version need to be bumped? I'm not sure if this is a breaking change.

Reply:
I would think so.  If the API used to return a DefaultError and it no longer does, clients that expect DefaultError would break when using the new API.